benchmarks/conext-table3.cc

```diff
@@ -0,0 +1,90 @@
+// This benchmark reproduces the CoNEXT 2014 results found in "Table 3: Space efficiency
+// and construction speed." It takes about four minutes to run.
+//
+// Results:
+//
+// metrics                                    CF     ss-CF
+// # of items (million)                   127.91    128.22
+// bits per item                           12.59     12.56
+// false positive rate                     0.19%     0.10%
+// constr. speed (million keys/sec)         2.06      1.88
+
+#include <climits>
+#include <iomanip>
+#include <vector>
+
+#include "cuckoofilter.h"
+#include "random.h"
+#include "timing.h"
+
+using namespace std;
+
+using namespace cuckoofilter;
+
+// The number of items sampled when determining the false positive rate
+const size_t FPR_SAMPLE_SIZE = 1000 * 1000;
+
+struct Metrics {
+  double ndv;    // # of items (million)
+  double space;  // bits per item
+  double fpr;    // false positive rate (%)
+  double speed;  // const. speed (million keys/sec)
+};
+
+template<typename Table>
+Metrics CuckooBenchmark(size_t ndv, const vector<uint64_t>& input) {
+  Table cuckoo(ndv);
+  auto start_time = Now();
+
+  // Insert until failure:
+  size_t inserted = 0;
+  while (inserted < input.size() and 0 == cuckoo.Add(input[inserted])) ++inserted;
```

Comment:
In keeping with Bin's plan to move everything to the Google C++ style guide, could you switch this (and elsewhere) to use the punctuation operators?  (&& instead of and)